maxim/logger/livekit/agent_activity.py

```
@@ -308,18 +325,50 @@ def handle_agent_speech_finished(self):
+
+    turn.usage = usage
+
 
 def pre_hook(self, hook_name, args, kwargs):
     ignored_hooks = ["push_audio"]
     try:
         if hook_name == "interrupt":
             get_thread_pool_executor().submit(handle_interrupt, self)
         elif hook_name == "_on_input_speech_started":
+            scribe().info(
+                f"[Internal][{self.__class__.__name__}] agent speech started handling"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)